docs/README.md

```diff
@@ -19,417 +17,498 @@
   </a>  
 </p>
 
-<p align="center">
-  <a href="#key-features">Key Features</a> •
-  <a href="#vision">Vision</a> •
-  <a href="#configuration">Configuration</a> •
-  <a href="#usage">Usage</a> •
-  <a href="#contributions">Contributions</a>
-</p>
+> 🤖 **Mergeable** helps automate your team's GitHub workflow without a single line of code.
 
-![screenshot](screenshot.gif)
+Some examples of what you can do:
 
+- Ensure Pull Requests follow conventions and [prevent merging of Pull Requests](#pull-requests).
+- [Notify author of failed guidelines](#issues) when opening an issue.
+- Schedule [detection stale issues and pull requests](#staleness) and notify author and collaborators.
+- And [more](#configuration)
 
-## Key Features
+---
 
-- [Prevent pull requests to be merged](#pull-requests)  based on configured rulesets
-- [Notify author of failed guidelines when openning an issue](#issues) based on configured rulesets
-- [Detect stale issues and pull requests](#staleness) and notify author and collaborators.
+**📖 Contents:** [Usage](#usage) ◦ [Configuration](#configuration) ◦ [Roadmap](#roadmap) ◦ [Support](#support) ◦ [Contributions](#contributions) ◦ [Authors](#authors)
 
-### Pull Requests
+---
 
-Validate pull requests for mergeability based on content and structure of your PR (title, labels, milestone, project, description, approvals, etc). The rule-sets are fully configurable. Here are a few examples of what you can do:
 
-- Prevent accidental merging of Pull Requests that are work in progress by labeling it `wip` or prefixing the title with the abbreviation.
+# Usage
 
-- Ensure all Pull Requests have a description so that when you view through history you still have context.
+1. [Install](https://github.com/apps/mergeable) the Mergeable GitHub App.
+2. [Create](#configuration) your recipe(s). Here are some [examples](#examples).
+3. Commit and push the recipes to your repository at `.github/mergeable.yml`
 
-- Ensure that all Pull Requests are approved by a specific list of users. It is especially useful if one of the users is not a collaborator of your repository -- something GitHub does not already support.
+> ☝ **NOTE:** You can also [deploy to your own server](deploy.md).
 
-- Ensure that all Pull Requests merged are in a specific GitHub Project. Mergeable even detects when you are closing an issue that is associated with the right project. This is very useful when your process includes QA validation of stories.
+# Configuration
 
-- [And more](#configuration).
+**Mergeable** is **highly** configurable.
+Define your recipes by creating a `.github/mergeable.yml` file in your repository.
```

Comment:
Would it make more sense to say `Define your own rule-set or use one of the recipes` something to that effect?

Reply:
I think that we don't have enough collection of existing examples to choose from. Hence I feel we can leave it this way. I did contemplate on using the word rule-set but that is the same as recipe...